client/src/modules/vouchers/toolkit/patient_fled/patient_fled.modal.html

```diff
@@ -0,0 +1,64 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>VOUCHERS.GLOBAL.PATIENT_FLED</li>
+    <li class="title" translate>VOUCHERS.GLOBAL.IMPORT_TRANSACTION</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" novalidate>
+  <div class="modal-body">
+    
+    <bh-account-select
+      name="account_id"
+      account-id="ToolCtrl.account.id"
+      on-select-callback="ToolCtrl.onSelectAccount(account)">
```

Comment:
This should be only expense accounts.  Can you use the [bhAccountSelect's `accountTypeId` option](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/js/components/bhAccountSelect.js#L11) to filter for only expense accounts?